README.md

```diff
@@ -148,7 +148,7 @@ If you use `http` protocol, the config section like this:
 
 ## Programmatic API
 
-When you require `mention-bot` you will get all the functions exposed by `mention-bot.js` module. You are expected to manage your own server and also connection to the github repository.
+When you require `mention-bot` you will get all the functions exposed by [`mention-bot.js`](https://github.com/facebook/mention-bot/blob/master/mention-bot.js) module. You are expected to manage your own server and also connection to the github repository.
```

Comment:
Sure! I just made an update to the [PR](https://github.com/facebook/mention-bot/pull/116/files) to make it fit for the merge :)